docs/css/remove.md

```diff
@@ -1,4 +1,34 @@
 ## Removing `sanitize.css`
 
-Delete [line 19 in `app.js`](../../app/app.js#L19) and remove it
-from the `dependencies` in [`package.json`](../../package.json)!
+To remove `sanitize.css` you will need to remove it from both:
+- [`app.js`](../../app/app.js)
+```diff
+import FontFaceObserver from 'fontfaceobserver';
+import { useScroll } from 'react-router-scroll';
+-import 'sanitize.css/sanitize.css';
+
+// Import root app
+import App from 'containers/App';
+```
+
+- [`package.json`](../../package.json)!
```

Comment:
The term "Canonical" still isn't in my vocabulary, but does this mean following the instructions on [this page](https://help.github.com/articles/getting-permanent-links-to-files/) to get a permanent link???

This does sound useful in examples that link to a specific line, but does this mean the link will need updating with each release that changes the linked file?